atom/browser/native_window_mac.mm

```diff
@@ -962,13 +962,14 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   // We will manage window's lifetime ourselves.
   [window_ setReleasedWhenClosed:NO];
 
-  // Hide the title bar.
-  if (title_bar_style_ == HIDDEN_INSET || title_bar_style_ == HIDDEN) {
+  // Hide the title bar background
+  if (title_bar_style_ != NORMAL) {
     if (base::mac::IsAtLeastOS10_10()) {
       [window_ setTitlebarAppearsTransparent:YES];
```

Comment:
Should this be `titleBarAppearsTransparent` instead of `setTitleBarAppearsTransparent`? NSWindow has titleBarAppearsTransparent https://developer.apple.com/documentation/appkit/nswindow/1419167-titlebarappearstransparent?language=objc. I could not find anything in the Apple documentation for the term setTitleBarAppearsTransparent.

Reply:
The set prefix is because we aren't addressing the window itself rather a wrapper.

https://github.com/electron/electron/blob/daad465fcdf4f3d861f1cce9a08d97a6fe3639fc/atom/browser/native_window_mac.h#L177

Reply:
Ah! Now that I think of it, the code would not have compiled if an undefined method was used.